src/neo_omnidrive_socketcan.cpp

```diff
@@ -133,6 +136,7 @@ class NeoSocketCanNode {
 		}
 		m_node_handle.param("request_status_divider", m_request_status_divider, 10);
 		m_node_handle.param("heartbeat_divider", m_heartbeat_divider, 10);
+		m_node_handle.param("diagnostics_divider", m_diagnostics_divider, 20);
```

Comment:
the default value could be changed to `0` which would mean "don't publish diagnostics"